docs/releases/3.1.txt

```diff
@@ -70,8 +70,18 @@ JSONField for all supported database backends
 Django now includes :class:`.models.JSONField` and
 :class:`forms.JSONField <django.forms.JSONField>` that can be used on all
 supported database backends. Both fields support the use of custom JSON
-encoders and decoders. The model field supports the introspection, lookups, and
-transforms that were previously PostgreSQL-only.
+encoders and decoders. The model field supports the :ref:`introspection,
+lookups, and transforms <querying-jsonfield>` that were previously
+PostgreSQL-only::
```

Comment:
I would add `querying-jsonfield` reference with an example.

Reply:
Can you elaborate? Sorry, I'm not sure what you mean here. I have a reference to `querying-jsonfield` already?

I wanted to create a small, condensed example snippet that showed off a couple of the querying features - I went for a basic equality query combined with a `__contains` as these seem interesting.

Reply:
Sorry, typo it should be _"..reference **without** an example"_. But OK, let's leave an example. I added `has_key`.

Reply:
Awesome, it looks a lot better. Thanks!

I think code samples are like pictures to developers - they tell a thousand words (and our eyes are drawn to them), and we should definitely include them for the big-ticket features where we can 💪